src/utils/colonyActions.ts

```diff
@@ -56,6 +56,13 @@ export const getValuesForActionType = (
         fromDomain: argsObj.domainId,
       };
     }
+    case ColonyActions.MoveFunds: {
+      return {
+        amount: argsObj.amount,
+        fromDomain: argsObj.fromPot,
+        toDomain: argsObj.toPot,
+      };
```

Comment:
Please also return the initiator from the agent:

```suggestion
      return {
        amount: argsObj.amount,
        fromDomain: argsObj.fromPot,
        toDomain: argsObj.toPot,
        initiator: argsObj.agent,
      };
```